src/domains/functions/function_admin.tf

```diff
@@ -299,6 +299,96 @@ module "function_admin_staging_slot" {
   tags = var.tags
 }
 
+// ----------------------------------------------------
+// Alerts
+// ----------------------------------------------------
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "alert_failed_delete_procedure" {
+  enabled             = true
+  name                = "[IO-AUTH | ${module.function_admin.name}] Found one or more failed DELETE procedures"
+  resource_group_name = azurerm_resource_group.admin_rg.name
+  scopes              = [data.azurerm_application_insights.application_insights.id]
+  description         = <<EOT
+    Found one or more failed DELETE procedures.
+    Check ${local.function_admin.app_settings_common.FAILED_USER_DATA_PROCESSING_TABLE} table in
+    ${data.azurerm_storage_account.storage_api.name} storage account for more details
+    EOT
+
+  severity                = 1
+  auto_mitigation_enabled = false
+  location                = azurerm_resource_group.admin_rg.location
+
+  // check once every minute(evaluation_frequency)
+  // on the last minute of data(window_duration)
+  evaluation_frequency = "P1D"
+  window_duration      = "P1D"
```

Comment:
Every minute or every day? Either fix the comment or the settings.
(In my opinion a daily check is enough)

Reply:
changed comment in https://github.com/pagopa/io-infra/pull/1334/commits/8371ddcb2dc3d010f49f5c9ff89b8bb95e33343f